Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd

```diff
@@ -5548,7 +5548,7 @@ least 1 entry in RX FIFO during this time, this bit is clear by write 1</descrip
      <dim>2</dim>
      <dimIncrement>4</dimIncrement>
      <name>MEMBADDR[%s]</name>
-     <description>Memory Base Address register. This is the base address of the addressable memory region in this microcontroller’s RAM. Because the addressable memory is mapped in 16M boundaries, the lower 24 bits are fixed to 0.</description>
+     <description>Memory Base Address register. This is the base address of the addressable memory region in this microcontrollers RAM. Because the addressable memory is mapped in 16M boundaries, the lower 24 bits are fixed to 0.</description>
```

Comment:
's was correct.

Reply:
Had to revise this description. The ' was not supported in SVDConv.